modules/dailymotionBidAdapter.md

```diff
@@ -55,16 +56,53 @@ const adUnits = [
       video: {
         context: 'instream',
         playerSize: [1280, 720],
+        api: [2,7],
+        description: 'this is a video description',
+        duration: 556,
+        iabcat2: 'test_cat',
+        id: '54321',
+        lang: 'FR',
+        private: false,
+        tags: 'tag_1,tag_2,tag_3',
+        title: 'test video',
+        topics: 'topic_1, topic_2',
+        xid: 'x123456'
       },
     },
     bids: [{
       bidder: "dailymotion",
-      params: {}
+      params: {
+        video: {
+          description: 'this is a test video description',
+          duration: 330,
+          iabcat2: 'test_cat',
+          id: '54321',
+          lang: 'FR',
+          private: false,
+          tags: 'tag_1,tag_2,tag_3',
+          title: 'test video',
+          topics: 'topic_1, topic_2, topic_3',
+          xid: 'x123456'
+        }
+      }
     }]
   }
 ];
 ```
 
+Following video metadata fields can be added in mediaTypes.video or bids.params.video. If a field exists in both places, it will be overridden by bids.params.video.
+
+* `description` - Video description
+* `duration` - Video duration in seconds
+* `iabcat2` - Video IAB category
+* `id` - Video unique ID in host video infrastructure
+* `lang` - Main language used in the video
```

Comment:
@Aditi0101, @jeremy-le-dev, @rumesh should we specify ISO code for lang? (ISO 639)

Reply:
Good point, yes IMO we should add this information 👍🏻